src/components/Account/Account.tsx

```diff
@@ -8,6 +8,8 @@ import { useMutation } from 'react-query';
 import styles from './Account.module.scss';
 
 import type { FormSectionContentArgs, FormSectionProps } from '#components/Form/FormSection';
+import type { Consent } from '#types/account';
+import { REGISTER_FIELD_VARIANT } from '#src/services/inplayer.account.service';
```

Comment:
We should avoid having imports from services/stores inside components. What do you think of creating a new file under utils/registerFields.ts and moving `REGISTER_FIELD_VARIANT` there?
Same goes for `ConsentFieldVariants`.